.cbmc-batch/source/proof_allocators.c

```diff
@@ -16,14 +16,16 @@
 #include "proof_allocators.h"
 #include <stdlib.h>
 
-//Use the same functions as the standard default allocator, but nondeterministically have malloc
-//return null.  This is needed because the CBMC model of malloc cannot fail, so we cannot
-//test the null case otherwise.
+// Use the same functions as the standard default allocator, but nondeterministically have malloc
```

Comment:
We should be able to just use the ones from c-common

Reply:
True, I'm not actually sure why we have this local copy. I'll remove it.